src/script/script.cpp

```diff
@@ -238,11 +240,11 @@ bool CScript::IsPushOnly() const
     return true;
 }
 
-std::string CScript::ToString() const
```

Comment:
Just for consistency within the file. That method was the only one using the prefix in that file because of the "using namespace std" up on line 12. That's all I was thinking when I did that.